parsel/utils.py

```diff
@@ -82,7 +82,19 @@ def extract_regex(regex, text, replace_entities=True):
         return strings
     return [w3lib_replace_entities(s, keep=['lt', 'amp']) for s in strings]
 
-def shorten_selector_data(data, width):
-    """Shortens the preview of extracted data by adding placeholder '...'
-    """
-    return data[:width - 3] + "..." if (len(data) > width - 3) else data
\ No newline at end of file
+
+# Note: This suffix is used in __repr__ return values, use an ASCII suffix as
+# long as Python 2 support is needed.
+_SHORTENING_SUFFIX = '...'
```

Comment:
What do you think about making this an argument to shorten function (with a default value), and dropping _SHORTENING_LENGTH constant, as `len(text)` is super-fast, no need to cache it?